src/third_party/cppcodec/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-cmake_minimum_required(VERSION 2.8.5)
-project(cppcodec CXX)
-set(PROJECT_VERSION 0.1)
+cmake_minimum_required(VERSION 2.8.12)
+cmake_policy(SET CMP0048 NEW)
```

Comment:
It seems that [CMP0048](https://cmake.org/cmake/help/latest/policy/CMP0048.html) was introduced in cmake 3.0. Does that mean that the cmake_minimum_required should be higher?